view/frontend/web/js/pcjs.js

```diff
@@ -229,11 +228,12 @@ define(['jquery', 'Magento_Swatches/js/swatch-renderer', 'priceBox'], function (
                         var swatchOptions = $item.find(".swatch-opt");
                         $(swatchOptions).each(function () {
                             var option = $(this);
+                            console.log(option.data().pureclarityJsonconfig);
                             var jsonConfig = option.data().pureclarityJsonconfig;
                             var swatchRenderJson = option.data().pureclaritySwatchrenderjson;
                             if (jsonConfig && swatchRenderJson) {
                                 swatchRenderJson.numberToShow = pureclarityConfig.swatchesToShow;
-                                option.SwatchRenderer(swatchRenderJson);
+                                window.pureclarityConfig.swatchRenderer(swatchRenderJson);
```

Comment:
Prevents issue on earlier versions of Magento where path of swatch renderer has been changed, now set in configuration